### PR TITLE
#37620 Added nodetype and nodeids on task creation and response

### PR DESCRIPTION
**Pull request by bolleb**

### CODE REVIEW

File: iot/grpcapi.proto [hunk 1]
```diff
@@ -58,6 +58,8 @@ message TaskDescription {
   TaskStatus status = 8;
   int64 status_updated_at = 10; // ms
   int64 long_id = 11;
+  string nodeType = 12;
```
Comment (j0s) [inline iot/grpcapi.proto]: To follow the style of the rest of the file, these should probably be `node_type` and `node_ids`.

File: iot/grpcapi.proto [hunk 2]
```diff
@@ -68,6 +70,8 @@ message InitialTaskDescription {
   FunctionalLocationIds functional_location_ids = 5;
   string external_task_id = 6;
   TaskStatus status = 7;
+  string nodeType = 8;
```
Comment (j0s) [inline iot/grpcapi.proto]: Same as above.

File: iot/grpcapi.proto (hunk 1 above)
Comment (bolleb) [reply iot/grpcapi.proto]: Tack

File: iot/grpcapi.proto (hunk 2 above)
Comment (bolleb) [reply iot/grpcapi.proto]: Fixat

**Pull request merged**